### PR TITLE
Keyboard shortcut works even when singleClick [L=C] is disabled

### LINKED ISSUE #3350: L+C keyboard shortcut doesn't work when L=C is hidden

### PR DESCRIPTION
**Pull request by erikdesjardins**
Fixes #3350 

(untested)

### CODE REVIEW

File: lib/utils/dom.js [hunk 1]
```diff
@@ -63,12 +63,6 @@ click.isProgrammaticEvent = e => {
 	return e.clientX === 1 && e.clientY === 1;
 };
 
-export function mousedown(obj, button = 0) {
```
Comment (andytuba) [inline lib/utils/dom.js]: Is this still used for clicking native reddit elements, like keyboard nav invoking upvote/downvote?

File: lib/modules/keyboardNav.js [hunk 2]
```diff
@@ -30,6 +29,7 @@ import * as SaveComments from './saveComments';
 import * as SelectedEntry from './selectedEntry';
 import * as ShowImages from './showImages';
 import * as ShowParent from './showParent';
+import * as SingleClick from './singleClick';
```
Comment (andytuba) [inline lib/modules/keyboardNav.js]: :+1:

File: lib/utils/dom.js (hunk 1 above)
Comment (erikdesjardins) [inline lib/utils/dom.js]: 😜 

image

s/a use/any future use/ to clarify

Comment (andytuba) [inline lib/utils/dom.js]: Oh, they use `click` instead of `mousedown`.

**Pull request merged**